tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>rename samples to <code>num_samples</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good idea.  I recently changed the Observation property from "samples" to "n_sample".  I should make the constructor arguments consistent.</p><p> </p><p>If you have feedback on the <code>n_sample</code>,  <code>n_local</code>  and other property names feel free to comment on the toast/observation.py file at the bottom where the <code>Observation</code> class is defined.</p>

Reply:
ok, please mark the conversations as resolved as you go through the modifications

Reply:
<p>Changed to <code>n_samples</code>  to be closer to the Observation properties <code>n_all_samples</code>  and <code>n_local_samples</code> .</p>